### PR TITLE
Update forum doc and release note

### PR DESCRIPTION
**Pull request by readwalkerchang**
This fork is done among our team(The engagement team in C&C) which can be reviewed and merged

### CODE REVIEW

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
 
 ## 2022
 
+### April 7th, 2022
+#### Improvements
+
+* We release the new layout of [Mendix Forum](/developerportal/community-tools/mendix-forum/). In this new layout, you can see the following: adjustments:
```
Comment (FvB) [inline content/en/docs/releasenotes/developer-portal/_index.md]: We introduced a new layout of [Mendix Forum](/developerportal/community-tools/mendix-forum/). This layout features the following changes:

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+  * A side-bar navigation
```
Comment (FvB) [inline content/en/docs/releasenotes/developer-portal/_index.md]: * moved the main navigation to the left sidebar
* dedicated for page for your bookmarks
* breadcrumb navigation
* dedicated pages for forum content you've interacted with (asked, answered & commented for questions - submitted & commented for ideas)

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 3]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+  * A bookmark page
```
Comment (FvB) [inline content/en/docs/releasenotes/developer-portal/_index.md]: remove this line

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 4]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+  * A breadcrumb
```
Comment (FvB) [inline content/en/docs/releasenotes/developer-portal/_index.md]: remove this line

File: content/en/docs/developerportal/community-tools/mendix-forum.md [hunk 5]
```diff
@@ -162,19 +161,19 @@ All the submitted ideas are publicly visible, and each idea has a descriptive ti
...
-* The category (**6**)
+* The category (**3**)
 	* This is set when submitting an idea (for details, see the [Submitting an Idea](#submitting-an-idea) section)
-* Tags (**7**) (for example, **datagrid**, **listen-target**, **conditional-visibility**)
-* The status of the idea (**8**) (for descriptions of the various statuses, see the [Idea Details Page](#IdeaDetails) section)
-* When the idea was created (**9**)
-* The name, avatar, and level of the Mendix community member who created the idea (**10**)
-
+* Tags (**4**) (for example, **Native-widgets**)
+  * You can add more tags as you want
```
Comment (FvB) [inline content/en/docs/developerportal/community-tools/mendix-forum.md]: You can search for one or more tags

**Review by FvB — APPROVED** [review]

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**